onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -210,6 +210,7 @@ bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string
   auto model_proto = GetModelProtoFromFusedNode(graph_viewer);
 
   auto graph_proto = model_proto.mutable_graph();
+   std::string error_msg = "";
```

Comment:
This variable is not necessary.
You can simply do a: `throw "GPU plugin doesn't support... ";` for throw statements